packages/bot-skeleton/src/scratch/blocks/Binary/Trade Definition/trade_definition_tradeoptions.js

```diff
@@ -177,19 +192,28 @@ Blockly.Blocks.trade_definition_tradeoptions = {
             }
         }
     },
-    createPredictionInput(prediction_range) {
+    createPredictionInput(selected_trade_type_category, selected_trade_type, is_digit_trade_type_category) {
```

Comment:
@maryia-matskevich-deriv can we have a short discussion on the changes of this function? Just for clarification.

Reply:
sure